client/index.html

```diff
@@ -46,7 +46,7 @@
     <script type="text/javascript">
       window.CELLXGENE = {};
       window.CELLXGENE.API = {
-        prefix: `http://localhost:<%= CXG_SERVER_PORT %>${location.pathname}/api/`,
+        prefix: `http://localhost:<%= CXG_SERVER_PORT %>${location.pathname.replace(/\/$/, "")}/api/`,
```

Comment:
let's add a comment here to explain why we need to explicitly remove `/` and perhaps point to the Slack thread for future context!

Reply:
Slack threads aren't forever :/

Reply:
Oh right, not even the public channel.. good call. Maybe just copy some tl;dr and put it in the PR description as context?